agent/consul/state/intention_test.go

```diff
@@ -1199,6 +1199,70 @@ func TestStore_IntentionsList(t *testing.T) {
 	})
 }
 
+// TestStore_IntentionExact_ConfigEntries test that we can get a local config entry intention
+// and a peered config entry intention with an IntentionGetExact call
+func TestStore_IntentionExact_ConfigEntries(t *testing.T) {
+	s := testConfigStateStore(t)
+	inputs := []*structs.ServiceIntentionsConfigEntry{
+		{
+			Kind: structs.ServiceIntentions,
+			Name: "foo",
+			Sources: []*structs.SourceIntention{
+				{
+					Action: structs.IntentionActionAllow,
+					Name:   "bar",
+					Peer:   "peer1",
```

Comment:
You could set a unique `Description` field on each of these and assert below you get the correct one to be even more sure it works.

Reply:
✅